ci/test_wheel_cuproj.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -eou pipefail
+
+mkdir -p ./dist
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+RAPIDS_PY_WHEEL_NAME="cuproj_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
+
+# Install additional dependencies
+apt update
+DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends libgdal-dev
+python -m pip install --no-binary fiona 'fiona>=1.8.19,<1.9'
```

Comment:
Does cuproj need “fiona” or only cuspatial? Should it be listed as a test dependency for one or both packages?

Reply:
geopandas needs fiona.  But I think fiona doesn't work unless installed after gdal, and gdal has to be installed with apt.